pos_mail_receipt/static/src/js/screens.js

```diff
@@ -0,0 +1,99 @@
+/* License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+
+
+odoo.define("pos_mail_receipt.screens", function (require) {
+    "use strict";
+
+    var screens = require('point_of_sale.screens');
+    var rpc = require('web.rpc');
+
+    var ReceiptScreenWidget = screens.ReceiptScreenWidget.include({
+        renderElement: function() {
+            this._super();
+            var self = this;
+            this.$('.button.email').click(function(){
+                if (!self._locked) {
+                    self.email();
+                }
+            });
+        },
+        click_next: function() {
+            this._super();
+            this.$('.button.email').removeClass("highlight");
+        },
+        email: function() {
+            var self = this;
+            var email = false;
+            var body_from_ui = this.$('.pos-receipt-container').html()
+            if( this.pos.get_order().get_client() && this.pos.get_order().get_client().email ) {
+                self._send_email_server(this.pos.get_order().name, {"email": this.pos.get_order().get_client().email, "body_from_ui": body_from_ui});
+            } else {
+                this.gui.show_popup('textinput', {
+                    'title':_t('E-mail address to use'),
```

Comment:
Getting the error that `_t()` is in not found. It should be declared:
```js
odoo.define(...
    var core = require('web.core');
    var _t = core._t;
```

Reply:
Hi @mclaeysb  , thanks for the review :) Would you mind committing your fixes directly to the branch ? I think you have write access to it

Reply:
Sure! Sorry for the inconvenience..